### PR TITLE
Video thumbnail

### LINKED ISSUE #1790: Create a `VideoThumbnail` component

### PR DESCRIPTION
**Pull request by drillprop**
Fix #1790 
Design: https://www.figma.com/file/Pf31tuYpozYmpq163U2ho8/Web-Components?node-id=3629%3A119375
Just FYI didn't apply this component in VideoTile yet, I'll do it when I start working on https://github.com/Joystream/atlas/issues/1553

### CODE REVIEW

**Review by rafalpawlow — CHANGES_REQUESTED** [review]
When I click on thumbnail, sometimes NFT pill is being highlighted. Maybe we could set `user-select: none` to Pill component?
https://user-images.githubusercontent.com/13706246/147603282-6894b8d6-54cb-4e5f-b3ca-fabdbe4ae7b9.mov

File: src/components/_video/VideoThumbnail/VideoThumbnail.styles.ts [hunk 1]
```diff
@@ -0,0 +1,135 @@
...
+
+const sharedOverlayStyles = css`
+  position: absolute;
+  top: 0;
+  left: 0;
+  bottom: 0;
+  right: 0;
+  width: 100%;
+  height: 0;
+  padding-top: 56.25%;
```
Comment (rafalpawlow) [inline src/components/_video/VideoThumbnail/VideoThumbnail.styles.ts:14]: Wouldn't `height 100%` be enough since we have position absolute here?

**Review by rafalpawlow — CHANGES_REQUESTED** [review]

File: src/components/_video/VideoThumbnail/VideoThumbnail.styles.ts (hunk 1 above)
Comment (drillprop) [reply src/components/_video/VideoThumbnail/VideoThumbnail.styles.ts:14]: I refactored this a bit, but I don't think I could just get rid of this padding-top if that's what you mean

**Comment by drillprop** [comment]
> When I click on thumbnail, sometimes NFT pill is being highlighted. Maybe we could set `user-select: none` to Pill component? https://user-images.githubusercontent.com/13706246/147603282-6894b8d6-54cb-4e5f-b3ca-fabdbe4ae7b9.mov

Should be fixed. I didn't set `user-select` on Pill, I set this to its container(SlotContainer) in the video thumbnail.

I added some improvements as well. Now if we don't want to make the slot clickable, we can pass this as an option.  
This is will prevent situations when you click for example the pill and lifting animation don't start. Check out the video below: 



https://user-images.githubusercontent.com/51168865/147649976-81666408-292e-4ee9-9c24-c2b15f85e3cb.mov

**Comment by rafalpawlow** [comment]
@drillprop  Now the whole thumbnail is getting highlighted 😛

https://user-images.githubusercontent.com/13706246/147748525-bc8a1221-4e00-4b3c-99a4-2b351204b701.mov

**Review by rafalpawlow — APPROVED** [review]
After a discussion with @drillprop on rocket chat we decided that highlighting on click is not really an issue because user will not be able to click multiple times on the same tile.

**Pull request merged**